datacenter/dtr/2.2/guides/admin/configure/garbage-collection.md

```diff
@@ -0,0 +1,97 @@
+---
+description: Configure garbage collection in Docker Trusted Registry
+keyworkds: docker, registry, garbage collection, gc, space, disk space
+title: Docker Trusted Registry 2.2 Garbage Collection
+---
+
+## TL;DR
+
+1. Garbage Collection (GC) reclaims disk space from your storage by deleting
+unused layers
+2. GC can eb configured to run automatically with a cron schedule, and can also
+be run manually; only admins can configure these
+3. When GC runs DTR will be placed in read-only mode; pulls will work but pushes
+will fail.
+4. The UI will show when GC is running, and an admin can stop GC within the UI
+
+**Important notes**
+
+The GC cron schedule is set to run in **UTC time**.  Containers typically run in
+UTC time (unless the system time is mounted), therefore remember that the cron
+schedule will run based off of UTC time when configuring.
+
+GC puts DTR into read only mode; pulls succeed while pushes fail.  This is due
+to the stop the world mark-and-sweep over the blob store.  Pushing while GC runs
+may lead to undefined behaviour and data loss, therefore this is disbaled for
+safety.  For this reason it's generally best practice to ensure GC runs in the
+early morning on a Saturday or Sunday night.
+
+
+## Setting up garbage collection
+
+You can set up GC if you're an admin by hitting "Settings" in the UI then
+choosing "Garbage Collection".  By default, GC will be disabled, showing this
+screen:
+
+![](images/gc-default.png)
```

Comment:
Can you rename the images to match the article name too? like `garbage-collection-1.png`.
Also, the path for the images should be `../../images/garbage-collection-1.png`